moveit_kinematics/ikfast_kinematics_plugin/templates/ikfast61_moveit_plugin_template.cpp

```diff
@@ -780,6 +780,18 @@ bool IKFastKinematicsPlugin::searchPositionIK(const geometry_msgs::Pose& ik_pose
                           options);
 }
 
+// struct for storing and sorting solutions
+struct LimitObeyingSol
+{
+  std::vector<double> value;
+  double dist_from_seed;
+
+  bool operator<(const LimitObeyingSol& a) const
+  {
+    return dist_from_seed < a.dist_from_seed;
+  }
+};
+
```

Comment:
Please move this struct to the top of the file where the other auxiliary structures reside.